examples/nimble_gatt/Makefile

```diff
@@ -14,8 +14,8 @@ USEMODULE += nimble_svc_gatt
 
 # Use automated advertising
 USEMODULE += nimble_autoadv
-CFLAGS += -DNIMBLE_AUTOADV_DEVICE_NAME='"NimBLE GATT Example"'
-CFLAGS += -DNIMBLE_AUTOADV_START_MANUALLY=1
+CFLAGS += -DCONFIG_NIMBLE_AUTOADV_DEVICE_NAME='"NimBLE GATT Example"'
```

Comment:
Better put the renaming into separate commits.

Reply:
Yeah, bot really sure why I renamed...  I think to align with `autoconn` and other modules, but better to leave as it was (except for the CONFIG_ prefix)

Reply:
Ahh I guess I also wanted to align the file name...

Reply:
Did split.